docs/content/latest/api/ysql/exprs/func_currval.md

```diff
@@ -4,7 +4,7 @@ headerTitle: currval()
 linkTitle: currval()
 description: Returns the last value returned by the nextval() function for the specified sequence in the current session.
 menu:
-  latest:
```

Comment:
This should be latest. You need to edit the corresponding stable file:  `/docs/content/stable/api/ysql/exprs/func_currval.md`